app/uk/gov/hmrc/cataloguefrontend/platforminitiatives/PlatformInitiativesListPage.scala.html

```diff
@@ -0,0 +1,143 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.mvc.Call
+@import uk.gov.hmrc.cataloguefrontend.DateHelper._
+@import uk.gov.hmrc.cataloguefrontend.platforminitiatives._
+@import uk.gov.hmrc.cataloguefrontend.util.MarkdownLoader
+@import uk.gov.hmrc.cataloguefrontend.ViewMessages
+@import helper._
+@import scala.concurrent.Future
+@import java.net.URL
+@import scala.util.Try
+
+@this(viewMessages: ViewMessages)
+
+@(initiatives          : Seq[PlatformInitiative],
+displayChart           : Boolean,
+displayProgress        : Boolean,
+form                   : Form[_]
+)(implicit messages    : Messages, request: Request[_])
+
+@standard_layout(s"Platform Initiatives", "platforminitiatives") {
+<header>
+    <h1>Platform Initiatives</h1>
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+</header>
+
+@if(form.hasErrors) {
+<div class="alert alert-danger" role="alert">
+    <ul class="list">
+        @form.errors.map { error =>
+        <li class="alert-danger"> @Messages(error.message, error.args: _*)</li>
+        }
+    </ul>
+</div>
+}
+
+<div id="platform-initiatives">
+    <form action="/platform-initiatives" method="get">
+        <div class="col-xs-1 padding-reset-right">
+            <label for="search">Search:</label>
+        </div>
+        <div class="col-xs-5">
+            <input class="search form-control" id="search" type="text" name="initiativeName"
+                   value='@form("initiativeName").value'>
+        </div>
+        <div class="right" style="line-height:2.2em;">
+            <a href="?chart=true&progress=false">Pie chart</a> |
+            <a href="?chart=false&progress=true">Progress bar</a>
```

Comment:
you could use reverse routing to provide the url with compilation safety (parm naming etc) e.g. `platformInitiatives(chart = false, progress = true)`
again could be simplified with a single two-state param type